setup/management.sh

```diff
@@ -15,7 +15,19 @@ apt_install build-essential libssl-dev libffi-dev python3-dev
 # Install other Python 3 packages used by the management daemon.
 # The first line is the packages that Josh maintains himself!
 # NOTE: email_validator is repeated in setup/questions.sh, so please keep the versions synced.
-hide_output pip3 install --upgrade setuptools
+
+# the below fixes the issue with an older version of pip3 and setup tools
+# breaking pip3 via a race condition
+# see https://github.com/pypa/pip/issues/4253
+# and
+# https://github.com/mail-in-a-box/mailinabox/issues/1086
+
+# need to use hide output, because we have to re-install this.
+hide_output apt-get install -y --reinstall python-pkg-resources
```

Comment:
Hard to say. 

Each installation may have a different version of setuptools, depending on when they last installed the cryptography library. 

My guess is that the right way to do this is to lock libraries at specific versions and use virtualenv or something similar. 

I'm not tremendously familiar with the python ecosystem.